src/Crudit/Config/MailCrudConfig.php

```diff
@@ -90,26 +90,16 @@ public function getFields($key): array
 
     public function getItemActions(): array
     {
-        $actions = [];
-
-        $actions[] = ItemAction::new(
-            'action.show',
-            $this->getPath(CrudConfigInterface::SHOW),
-            Icon::new('search')
-        )->setCssClass('btn btn-primary btn-sm mr-1');
+        $actions = parent::getItemActions();
+        unset($actions[1]);
```

Comment:
be careful with that

because if the base getItemActions changes, your method will probably be wrong